lib/WebService/MinFraud/Record/Email.pm

```diff
@@ -48,6 +53,11 @@ This class contains the data for the email associated with a transaction.
 
 This class provides the following methods:
 
+=head2 first_seen
+
+A date string (e.g. 2017-04-24) to identify the date an email address was first
+seen.  This is expressed using the ISO 8601 date format.
```

Comment:
Maybe "by MaxMind" or something like that?

Also, channeling my inner Dave, one space after the period.